lang/ftidy/Portfile

```
@@ -25,63 +26,29 @@ long_description \
+          sha1   82da71cac7380016233ab2aa217dbf3a0dd3686f \
+          rmd160 1b96ed6a8e0ad787bdd46c21fbded3f445df7336 \
+          sha256 b08d174bdaf16baa2d6b614115f3b568587c9e635392899a8d6cc9cf1828a98c \
+          size   133309
 
-patchfiles              tidy.for.patch
+patchfiles       tidy.for.patch
 
-if {![variant_isset gcc43] && ![variant_isset gcc44] && ![variant_isset gcc45] && ![variant_isset gcc46] && ![variant_isset gcc47] && ![variant_isset gcc48] && ![variant_isset gcc49] && ![variant_isset gcc5] && ![variant_isset gcc6] && ![variant_isset gcc7]} {
-    default_variants    +gcc6
-}
-
-variant gcc43 conflicts gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc43} {
-    configure.compiler  macports-gcc-4.3
-}
-
-variant gcc44 conflicts gcc43 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc44} {
-    configure.compiler  macports-gcc-4.4
-}
-
-variant gcc45 conflicts gcc43 gcc44 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc45} {
-    configure.compiler  macports-gcc-4.5
-}
-
-variant gcc46 conflicts gcc43 gcc44 gcc45 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc46} {
-    configure.compiler  macports-gcc-4.6
-}
-
-variant gcc47 conflicts gcc43 gcc44 gcc45 gcc46 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc47} {
-    configure.compiler  macports-gcc-4.7
-}
-
-variant gcc48 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc49 gcc5 gcc6 gcc7 description {Build using gcc48} {
-    configure.compiler  macports-gcc-4.8
-}
-
-variant gcc49 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc5 gcc6 gcc7 description {Build using gcc49} {
-    configure.compiler  macports-gcc-4.9
-}
-
-variant gcc5 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc6 gcc7 description {Build using gcc5} {
-    configure.compiler  macports-gcc-5
-}
-
-variant gcc6 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc7 description {Build using gcc6} {
-    configure.compiler  macports-gcc-6
-}
-
-variant gcc7 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 description {Build using gcc7} {
-    configure.compiler  macports-gcc-7
-}
+compilers.choose f77
+compilers.setup  require_fortran
+use_configure    no
 
-build.pre_args        ${configure.fflags} ${configure.f77_archflags}
+build.pre_args   ${configure.fflags} ${configure.f77_archflags}
 pre-build {
-    build.cmd         ${configure.f77}
+    build.cmd    ${configure.f77}
 }
 
 destroot {
-    xinstall -m 755 -W ${worksrcpath} ftidy ${destroot}${prefix}/bin
+    xinstall -m 755 -W ${worksrcpath} ftidy       ${destroot}${prefix}/bin
+    xinstall -m 755 -W ${worksrcpath} tidy.ini    ${destroot}${prefix}/bin
+    xinstall -d                                   ${destroot}${prefix}/share/doc/ftidy
+    xinstall -m 644 -W ${worksrcpath} read1st.txt ${destroot}${prefix}/share/doc/ftidy/
+    xinstall -m 644 -W ${worksrcpath} tidy.txt    ${destroot}${prefix}/share/doc/ftidy/
```

The whole point of the `-W` option is that it lets you install multiple files from one source directory into a target directory:
```suggestion
    xinstall -d ${destroot}${prefix}/share/doc/ftidy
    xinstall -m 0644 -W ${worksrcpath} read1st.txt tidy.txt ${destroot}${prefix}/share/doc/ftidy/
```
I also recommend always using the `0` prefix on octal numbers. It makes no difference to Tcl but makes it clear to readers that the number is interpreted as octal, not decimal.

---

OK